x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -379,6 +395,7 @@ export interface LegacyEndpointEvent {
   event?: {
     action?: string;
     type?: string;
+    category?: string | string[];
```

Comment:
:grey_question: In what case would an event _not_ have a category?

Reply:
I'd have to dig into the endgame encoding library that converts the endgame events to ecs events. From here: https://github.com/elastic/endgame-evecs/blob/b75e97fbf77a49f34b8ac95048ea00ca73472e66/internal/events/encoder.go#L853 it looks like it's possible that the category won't be populated. I'm not sure I can give a good answer for the scenario around why it's not populated in that case. It probably should be populated in all cases but I'm trying guard us from thinking that it will _always_ be there.